docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,318 @@
+
+## Deploying an OpenShift Control Plane using IPI
+
+To stand up only a control plane with the IPI installer, the only change that needs to be made is to scale the compute nodes to zero. This can easily be done in the install config by changing the number of replicas in the compute machine pool to zero. This minimal example will show you how to do this. Consult the [cusomization documentation](machine-pool-customizations) for more detailed information about what changes to the machinepools are available.
+
+```yaml
+piVersion: v1
+baseDomain: example.com
+controlPlane:
+  name: master
+  replicas: 3
+compute:
+- name: worker
+  replicas: 0
+metadata:
+  name: test-cluster
+platform:
+  openstack:
+    apiFloatingIP: 128.0.0.1
+    cloud: mycloud
+    defaultMachinePlatform:
+      type: m1.s2.xlarge
+    externalNetwork: external
+pullSecret: '{"auths": ...}'
+sshKey: ssh-ed25519 AAAA...
+```
+
+## Deploying an OpenShift Control Plane using UPI
+
+Deploying a control plane in UPI also requires little to no modifications from your normal install workflow. To avoid unnessary duplication, the steps to stand up an OpenShift control plane won't be detailed here. Instead, follow the [UPI install documentation](install_upi.md). Since workers are deployed manually in UPI, you can simply choose not to deploy compute nodes. If you are using our ansible scripts, to deploy only a control-plane simply skip running the `compute-nodes.yaml` script.
+
+## Preparing your OpenShift Cluster to use SR-IOV Networks
+
+Once your OpenShift control plane is running, you will need to install the SR-IOV Network Operator. To do so, you will need access to an account on your OpenShift cluster with `cluster-admin` privileges. Once you are logged into that account, follow [these steps](sriov-operator) to install the Operator.
+
+## Creating SR-IOV Networks for Compute Machines
+
+The SR-IOV networks that are attached to compute nodes need to be created before creating the nodes. Please reference the example of how to create radio and uplink provider networks in OpenStack.
+
+```sh
+# Create Networks
+openstack network create radio --provider-physical-network radio --provider-network-type vlan --provider-segment 120
+openstack network create uplink --provider-physical-network uplink --provider-network-type vlan --external
+
+# Create Subnets
+openstack subnet create --network radio --subnet-range <radio_network_subnet_range> radio
+openstack subnet create --network uplink --subnet-range <uplink_network_subnet_range> uplink
+```
+
+## Creating SR-IOV Compute Nodes in IPI
+
```

Comment:
It would be nice if we could start with  0 worker nodes. 
But it looks like it is not so straight forward to boot up an ipi cluster with 0 workers. 
a full section on how to boot up a 0 worker ipi control plane is probably in order

Reply:
There is something about limitation on making the control-plane scheduable

https://github.com/openshift/installer/blob/master/docs/user/openstack/install_upi.md#make-control-plane-nodes-unschedulable

Reply:
I am changing the documentation to only recommend this as a post install operation on a cluster that already has a working control plane and at least the minimum recommended workers as standard VM worker nodes.